File: Reporting/CS/ReportMainDemo/BaseForms/RibbonReportDesigner/OfficeInspiredMainForm.cs
using DevExpress.XtraReports.UI;
using DevExpress.XtraReports.UserDesigner;

namespace RibbonReportDesigner {
    public partial class OfficeInspiredMainForm : MainFormBase {
        public OfficeInspiredMainForm() {
            InitializeComponent();
        }

        public override XRDesignPanel ActiveXRDesignPanel { get { return reportDesigner1.ActiveDesignPanel; } }

        public override void OpenReport(XtraReport report) {
            reportDesigner1.OpenReport(report);
        }
    }
}

File: Reporting/VB/ReportMainDemo/BaseForms/RibbonReportDesigner/OfficeInspiredMainForm.vb
Imports DevExpress.XtraReports.UI
Imports DevExpress.XtraReports.UserDesigner

Namespace RibbonReportDesigner

    Public Partial Class OfficeInspiredMainForm
        Inherits MainFormBase

        Public Sub New()
            InitializeComponent()
        End Sub

        Public Overrides ReadOnly Property ActiveXRDesignPanel As XRDesignPanel
            Get
                Return reportDesigner1.ActiveDesignPanel
            End Get
        End Property

        Public Overrides Sub OpenReport(ByVal report As XtraReport)
            reportDesigner1.OpenReport(report)
        End Sub
    End Class
End Namespace
